client/landing/stepper/declarative-flow/plugin-bundle-flow.ts

```diff
@@ -209,14 +209,15 @@ const pluginBundleFlow: Flow = {
 					}
 
 					// Custom end of flow.
-					const endReturn = bundleStepsSettings[ pluginSlug ]?.endFlow?.( {
+					const settings = bundleStepsSettings[ pluginSlug ];
+					const endReturn = settings?.endFlow?.( {
 						intent,
 						storeType,
 						adminUrl,
 						dispatch,
 						exitFlow,
 					} );
-					if ( false !== endReturn ) {
+					if ( settings?.endFlow && false !== endReturn ) {
```

Comment:
While testing it with the Sensei bundle (https://github.com/Automattic/wp-calypso/pull/85309), I noticed an issue here. Fixed as part of this commit: https://github.com/Automattic/wp-calypso/pull/84886/commits/b5fa4e03ae80ba0cda44c7a7c7e932b0e1a510be

It should return earlier only **if the `endFlow`** exists and it doesn't return `false`.